### PR TITLE
Made mainSass config item optional

### PR DESCRIPTION
**Pull request by azamat-sharapov**
#5

### CODE REVIEW

File: bootstrap-sass-styles.loader.js [hunk 1]
```diff
@@ -82,8 +82,9 @@ module.exports = function (content) {
       return "@import \"" + path.join(pathToBootstrapSass, "stylesheets/bootstrap", partial) + "\";";
     }).join("\n");
 
-  var mainSass = config.mainSass;
-  source += addImportReturnDependency(this, config, "mainSass");
+  if(config.mainSass) {
```
Comment (justin808) [inline bootstrap-sass-styles.loader.js]: Spacing

**Comment by justin808** [comment]
1. Did you test both cases?
2. Update readme
3. one suggestion

**Comment by azamat-sharapov** [comment]
1. Sure
2. Updated
3. ?

Also added space after `if`

**Comment by justin808** [comment]
@azamat-sharapov I'm going to have this merged by this evening. I'm tuning up my example as well. I'm considering removing support for having no config file, to simplify the project.

Any need for the no-config option?

It seems that if you're not going to customize the bootstrap installation, then there's not a huge advantage in using the sass version. I.e., you might as well use the built CSS files.

Thoughts?

**Comment by justin808** [comment]
@azamat-sharapov Please see 

https://github.com/justin808/bootstrap-sass-loader-example/pull/2

Could you please try out your changes in the context of this branch. Thanks collaborating.

If you run `webpack` in that project, you can put in print statements in the loader (assuming you used `npm link`. You can also run `node-debug webpack` and put in a debugger statement.

**Comment by azamat-sharapov** [comment]
> Any need for the no-config option?

@justin808 absolutely! Please don't do it. I'm using your loader mainly because I can customize bootstrap. 

> Could you please try out your changes in the context of this branch. Thanks collaborating.

Sorry, our timezone seems to be very different, so I just got your messages. I see you already have fixed that, but I'll also try on my end and let you know, if I notice any problem.

**Comment by justin808** [comment]
I actually fixed things. Options for config or no-config, and config even allows the extract-text plugin.

**Comment by azamat-sharapov** [comment]
yes, I noticed that later, later :) Optional config files is the way to go, thanks.

**Pull request merged**